src/rdiff_backup/Security.py

```diff
@@ -237,47 +237,63 @@ def _set_allowed_requests(sec_class, sec_level):
             "restore.MirrorStruct.initialize_rf_cache",
             "restore.MirrorStruct.close_rf_cache",
             "restore.MirrorStruct.get_diffs", "restore.ListChangedSince",
-            "restore.ListAtTime", "backup.SourceStruct.get_source_select",
-            "backup.SourceStruct.set_source_select",
-            "backup.SourceStruct.get_diffs",
+            "restore.ListAtTime",
             "compare.RepoSide.init_and_get_iter",
             "compare.RepoSide.close_rf_cache", "compare.RepoSide.attach_files",
             "compare.DataSide.get_source_select",
             "compare.DataSide.compare_fast", "compare.DataSide.compare_hash",
-            "compare.DataSide.compare_full", "compare.Verify"
+            "compare.DataSide.compare_full", "compare.Verify",
+            # API < 201
```

Comment:
Are you introducing another version  for the API ?
Do you think it's necessary since 201 was not even releases.

Reply:
No, it's only 200 and 201. I'm unsure why you think it might be so...